image.yaml

```diff
@@ -140,6 +140,7 @@ modules:
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
                   ref: master
+          - path: modules
```

Comment:
This should have a name to distinguish it from other repositories.  A good candidate would be `image.modules`.